itest/src/edu/stanford/nlp/dcoref/DcorefExactOutputITest.java

```diff
@@ -106,11 +106,10 @@ private static void saveResults(String filename, Map<Integer, CorefChain> chains
     FileWriter fout = new FileWriter(filename);
     BufferedWriter bout = new BufferedWriter(fout);
 
-    List<Integer> keys = new ArrayList<>(chains.keySet());
-    Collections.sort(keys);
+    Map<Integer, CorefChain> sorted = new TreeMap<>(chains);
```

Comment:
I don't expect this to be as optimized as to Collections.sort.  Seems like an unnecessary change

Reply:
It's a test, anyhow reverted.